### ISSUE TITLE
Web Interfaces does not show devices nor positions in full screen

### ISSUE THREAD

**Issue by fjsosa** [tananaev/traccar-web#594]
Hi, I'm new with traccar (and also with ext.js) , I'm studing the code here (master branch) and I detected something that seems to be an error.
I've created some devices, I've assigned the devices to some users, but those devices are not showed in the device list, and the last known position is also missed in the map.
But if you intialize the browser with an small weight (I did this to test the mobile version) the devices are showed correctly, and the position in the map too.
I've tried to debug the js files trying to understand the system initialization process and all seems to be ok. I'm afraid I will need more training to determine if this is a bug or a problem with the configuration with my instance here.
Could you please confirm if this is an issue? 

There are some recent changes in the files:
main.js
mainController.js

Thank you in advance and congratulations for this great work!.

Fernando Sosa.

**Comment by tananaev** [comment]
Can you provide a video or some more detailed description of the problem?

Have you tried it on the last official release? Does it work correctly?

**Comment by fjsosa** [comment]
Hi Anton, thanks for your quick answer, I've tried the following steps today in order to reproduce the behavior with the latest official release:

1) Download version 3.14 from: https://github.com/tananaev/traccar/releases/download/v3.14/traccar-other-3.14.zip

2) Execute this version using java -jar tracker-server.jar conf/traccar.xml (with the default database)

3) I've added a device and it was showed correctly.

4) stopped the app, changed the conf/traccar.xml to use mysql (mysql 5.6)

5) I've created an empty schema using utf8 as default collation.

6) Execute the app again

7) Added a device, and now the device is not showed. But if I load the app in the browser with a small width the devices are showed.

I'm attaching 
a) the console output for the command  java -jar tracker-server.jar conf/traccar.xml
b) Screenshots
c) a mysql script with my database structure and data
d)logs and config files


[traccar-device-issue.zip](https://github.com/tananaev/traccar-web/files/1337755/traccar-device-issue.zip)
